### DISCUSSION TITLE
question about sendFiles reponse object values

### DISCUSSION THREAD

**Discussion by rdekay-sm** [pitchmuc/adobe_analytics_api_2.0#41] [Q&A]
In the docs, this is said to be a requests response object, in the the code it appears to be list. either way I cannot seem to get a json response body or header , i do get "<Response [200]>" but no other info. Am I missing something?

Would like to get the reponse body:
{
  "file_id": "f5adc939-1b84-4cb9-93e3-95fabbbf0d59",
  "visitor_group_id": "ingestion_0",
  "size": 3087001,
  "received_date": 1610133354,
  "rows": 22207,
  "invalid_rows": 0,
  "upload_name": "2020-11-16-49612609730681170165362469521669600849355363338967582082-49612609730681170165362471072415569182562128009126479234.csv.gz",
  "status": "File received, awaiting processing",
  "status_code": "UPLOADED",
  "processing_log": "Processing complete: 22207 rows will be submitted.  No invalid rows.\n",
  "idempotency_key": "f5adc939-1b84-4cb9-93e3-95fabbbf0d59"
}

**Comment by pitchmuc** [comment]
Hello @rdekay-sm,
Sorry for the delay, just got my child a 10 days ago. 
Quite hectic time to maintain a library. :D 
You are correct, I was returning the response code and not the json object returned. 

This will be corrected in the next version coming in the next few weeks. 
I hope it is OK for you to wait a little bit. 
I would like to gather several improvement into one release. 

A bug has been raised, if you need to have this info, I will post the fix [there.](#42)
